contracts/tokenomics/staking/src/contract.rs

```diff
@@ -63,7 +63,7 @@ pub fn instantiate(
     // Create token
     let sub_msg: Vec<SubMsg> = vec![SubMsg {
         msg: WasmMsg::Instantiate {
-            admin: None,
+            admin: Some(info.sender.to_string()),
```

Comment:
let's use msg.owner instead of info.sender, as usually we deploy from dev address and set owner to multisig

Reply:
Ok, done